static/bidder-info/onetag.yaml

```diff
@@ -15,6 +15,6 @@ capabilities:
       - native
 userSync:
   iframe:
-    url: "https://onetag-sys.com/usync/?redir={{.RedirectURL}}"
+    url: "https://onetag-sys.com/usync/?redir={{.RedirectURL}}&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}"
```

Comment:
I am unable to verify the user sync url. I tested this config for the Xandr identity server (see request below) and received a response of `{}` from the `onetag-sys.com` endpoint instead of an expected user sync script.

```
https://onetag-sys.com/usync/?redir=https%3A%2F%2Fib.adnxs.com%2Fprebid%2Fsetuid%3Fbidder%3Donetag%26gdpr%3D0%26gdpr_consent%3D%26f%3Db%26uid%3D%24%7BUSER_TOKEN%7D&gdpr=0&gdpr_consent=&us_privacy=
```

Reply:
Hi @SyntaxNode, have you tried adding a "Referer" header to the user sync request?
The requests without that header are marked as invalid from our endpoint.

Reply:
@onetag-dev we test userSync urls in browser and we only take the url and observe the network activity. We don't send any extra headers. We expect to see 2 or more requests returned, one of them should be 302 redirect to the "redir={{.RedirectURL}". 
Here is the url I built to test your change: 
https://onetag-sys.com/usync/?gdpr=&gdpr_consent=&us_privacy=&redir=https%3A%2F%2Fprebid.adnxs.com%2Fpbs%2Fv1%2Fsetuid%3Fbidder%3Dlogicad%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%24%7BUSER_TOKEN%7D

Reply:
@VeronikaSolovei9 please can you try again? Thanks

Reply:
It doesn't work. Request fails, there are no redirects. Please check it on your side. 
Here is a working example: 
https://cr-p31.ladsp.com/cookiesender/31?r=true&gdpr=&gdpr_consent=&ru=https%3A%2F%2Fprebid.adnxs.com%2Fpbs%2Fv1%2Fsetuid%3Fbidder%3Dlogicad%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%24UID
You can see a redirect and request to "setuid" endpoint with UID replaced with actual value.

Reply:
@onetag-dev Friendly reminder about this comment thread.

Reply:
@VeronikaSolovei9  @SyntaxNode I apologize for the delay: we should have made the necessary changes for the correct behaviour.

Reply:
Thank you, looks like it works correctly